qa/smoke-test-reindex-with-groovy/src/test/resources/rest-api-spec/test/reindex/10_script.yaml

```diff
@@ -0,0 +1,397 @@
+---
+"Modify a document":
+  - do:
+      index:
+        index:  twitter
+        type:   tweet
+        id:     1
+        body:   { "user": "kimchy" }
+  - do:
+      indices.refresh: {}
+
+  - do:
+      reindex:
+        refresh: true
+        body:
+          source:
+            index: twitter
+          dest:
+            index: new_twitter
+          script:
+            inline: ctx._source.user = "other" + ctx._source.user
```

Comment:
I've intentionally not ported master to use painless so this will match master exactly.